com.github.kmwallio.thiefmd.json

```diff
@@ -0,0 +1,106 @@
+{
+  "app-id":"com.github.kmwallio.thiefmd",
+  "runtime":"org.gnome.Platform",
+  "runtime-version":"3.36",
+  "sdk":"org.gnome.Sdk",
+  "command":"com.github.kmwallio.thiefmd",
+  "copy-icon": true,
+  "finish-args":[
+    "--share=ipc",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri",
+    "--socket=x11"
+  ],
+  "cleanup":[
+    "/cache",
+    "/man",
+    "/share/aclocal",
+    "/share/devhelp",
+    "/lib/systemd",
+    "/include",
+    "/lib/pkgconfig",
+    "/lib/libvala*",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/vala",
+    "/bin/vapi*",
+    "*.a",
+    "*.la"
+  ],
+  "modules":[
+    {
+      "name": "libgee",
+      "make-install-args": ["girdir=/app/share/gir-1.0", "typelibdir=/app/lib/girepository-1.0"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.1.tar.xz",
+        "sha256": "bb2802d29a518e8c6d2992884691f06ccfcc25792a5686178575c7111fea4630"
+      }]
+    },
+    {
+      "name": "granite",
+      "buildsystem": "meson",
+      "cleanup": ["/bin", "/share/applications"],
+      "sources": [{
+        "type": "archive",
+        "url": "https://github.com/elementary/granite/archive/5.4.0.tar.gz",
+        "sha256": "8194031cac3b87d84a3fab9c30270485ce73f8b8ec23f26c9152b6859c8a18fd"
+      }]
+    },
```

Comment:
Any reason for not using the elementary BaseApp?

Reply:
The vala compiler is a bit older in the elementary BaseApp. There's changes between 0.40.11 and 0.48.10 that seemed to prevent compilation.